diff.py

```diff
@@ -831,9 +856,28 @@ def do_diff(basedump, mydump):
                         branch_targets2[j1 + k] + ":", "~>"
                     )
 
+            if args.source and comments2[j1 + k] is not None:
+                out2 += f" {comments2[j1 + k][0]}"
```

Comment:
only do this if `has2`, otherwise this may produce incorrect results or crash

Are these comments useful? The image just shows unmangled function names

Reply:
Yeah, they're useful for function calls:

      2738:    bl      216c     <memcpy@plt> ~>       

Sometimes the function call appears in the source code, but not always:

![image](https://user-images.githubusercontent.com/4209061/84061080-957afd80-a9bd-11ea-9330-b639cb364772.png)

when the call is implicitly generated (when calling destructors of parent classes, or calling constructors for member variables for example).

Reply:
Ah, I see. For `bl` it might be nicer to parse out the comment similar to a relocation and present it as part of the asm (not just with --source), but for tail recursion that does get a bit trickier... That adrp comment looks vaguely useful as well